### PR TITLE
Sign in revamp to have the same look and feel of Designs

### PR DESCRIPTION
**Pull request by ahussein3**
- Design Link : https://www.figma.com/file/cZ3Zyyin4rRC7dLTBqK0eT/Datum---Brand%2C-Marketing-Website%2C-Design-Guide?type=design&node-id=837-8935&mode=design&t=jfMl7ClWAInBKngL-0

### CODE REVIEW

File: apps/operator/src/app/(auth)/login/page.tsx
Comment (hannahdatum) [inline apps/operator/src/app/(auth)/login/page.tsx:1]: Page components we should store in `/components/pages` to keep the app dir free of markup.

File: apps/operator/src/app/(auth)/login/page.tsx [hunk 1]
```diff
@@ -1,7 +1,6 @@
 'use client'
```
Comment (hannahdatum) [inline apps/operator/src/app/(auth)/login/page.tsx:1]: We should try and push the use of client components as far away as we can

File: apps/operator/src/app/(auth)/login/page.tsx [hunk 2]
```diff
@@ -11,9 +10,12 @@ import { TextInput } from '@repo/ui/text-input'
 import { MessageBox } from '@repo/ui/message-box'
 import { SimpleForm } from '@repo/ui/simple-form'
 import { type LoginUser } from '../../../lib/user'
-import logoReversed from '../../../../public/logos/logo_orange_icon.svg'
 import visibilityOff from '../../../../public/icons/visibility-off.svg'
 import visibility from '../../../../public/icons/visibility.svg'
+import LandingBackground from '../../../../public/backgrounds/landing-bg.svg'
```
Comment (hannahdatum) [inline apps/operator/src/app/(auth)/login/page.tsx:15]: These svgs don't need to be imported, the Image src can reference them directly without the `public` eg. `/backgrounds/landing-bg.svg`

File: apps/operator/src/app/(auth)/login/page.tsx [hunk 3]
```diff
@@ -64,113 +66,128 @@ const AuthLogin: React.FC = () => {
   }
 
   return (
-    <main className="flex flex-col min-h-screen w-full items-center space-between dark:bg-dk-surface-0 bg-surface-0">
+    <main className="flex flex-row h-full w-full items-center space-between bg-blackberry-900">
```
Comment (hannahdatum) [inline apps/operator/src/app/(auth)/login/page.tsx:69]: We'll be moving away from inline styles in favour of https://www.tailwind-variants.org/docs/slots and https://www.tailwind-variants.org/docs/variants. Style files will be colocated for a separation of concerns

File: apps/operator/src/app/(auth)/login/page.tsx [hunk 4]
```diff
@@ -64,113 +66,128 @@ const AuthLogin: React.FC = () => {
...
-              >
-                {showPassword ? (
-                  <Image  alt="hide password icon" src={visibilityOff} />
-                ) : (
-                  <Image alt="show password icon" src={visibility} />
-                )}
-              </button>
-            </div>
+              <div className="relative">
+                <TextInput
```
Comment (hannahdatum) [inline apps/operator/src/app/(auth)/login/page.tsx:87]: For forms we'll be using `react-hook-form` and validating with `zod`

**Review by matoszz — COMMENTED** [review]
One main thing: could the login form and image be inverted such that the login fields are on the right hand side and image is on the left?

**Comment by matoszz** [comment]
closing in lieu of @hannahdatum 's other PR including storybook

**Pull request closed without merging**